### ISSUE TITLE
[Item Unification] Items both unifying when disabled and converting incorrectly

### ISSUE THREAD

**Issue by UnlimatedStone9** [Blood-Asp/GT5-Unofficial#1512]
Hey there, I've been having an issue with the item unification feature, first off, I've been getting an issue where even though all item unification is disabled (in the MaterialProperties config file) items are still converting, not everything is converting however some items are, and I can ensure you I disabled every item in there

Another issues is that I'm getting items converting over when they most definitely are not the same item, for example the Terrafirmacraft chipped diamond, it's converting over into the Terrafirmacraft chipped ruby, which is both entirely illogical, and not any of GregTechs versions of its gems

I could really use assistance please, thank you!           -UnlimatedStone9

**Comment by UnlimatedStone9** [comment]
Possibly an overall disable feature could be added into a config file, simply have
EnableUnification_true=true/false ?
Just an idea that could be very useful, thanks!

**Comment by bHXDgiuMa5yLHyC8** [comment]
Try opening Unification.cfg and setting the TFC items to "true".

**Comment by UnlimatedStone9** [comment]
Why would setting them to true do anything? they were true by default, I don't want that to be the case

**Comment by UnlimatedStone9** [comment]
GregTech's TFC "Compatibility" only ever made things worse, adding items that would replace TFC's was not at all in any way helpful, it broke everything actually, because having items convert that can't be used in place of the originals was not useful at all, I am also aware it has worked before however right now I am having what I deem a bug, where an item from TFC is converting into another item from TFC, even when unification is disabled
I may be coming off as rude but trust me, I don't intend for that

**Comment by UnlimatedStone9** [comment]
Wait, I'm assuming "B:plateZinc_false=false" means that the item will convert over, thus setting it to true will disable the items converting into GregTech equivalents? if so then that's a little misleading, them being set to true would make me believe that they were converting over
Either way I can give that a go

**Comment by draknyte1** [comment]
Not our job to support mods that can’t add recipes properly to allow oredict. 

We DO NOT explicitly add any support for TFC, nor do we claim ANYWHERE they’re compatible. Google will tell you the exact opposite, If anything.. go complain to the TFC dev to develop his mod properly and then we can continue this discussion. /2cents

**Comment by UnlimatedStone9** [comment]
No, GregTech specifically added items to attempt to bridge compatibility with TFC, which overall caused more issues, this port here of GT still contains those features, the chipped gems etc from GT and TFC versions of metals existing in the mod ONLY as a means of bridging them, TFC is also not being developed at all anymore, just little maintainence, it is in the process of being ported to 1.12.2 also, however for now I am stuck with the 1.7 version
No it is not your job to support these mods however TFC doesn't tend to add oredict support for any of its items, which by the way are being converted over using GT mechanics
TFC Bismuth Bronze for example, the only other mod that adds that metal, other than GT converts into GT's metal, which did not exist prior to Gregs attempt to bridge the mods

**Comment by UnlimatedStone9** [comment]
I don't want to cause an argument or anything, I just wanted a little support is all, possibly assistance with how to disable the item unification which does not seem to work correctly for me

**Comment by UnlimatedStone9** [comment]
Also, going here from the links provided as your wiki for the mod: https://ftb.gamepedia.com/Mod_Compatibility_(GregTech_5), scrolling down explains that GT does add support for TFC's alloys and metals, (Not mentioned but the gem items too), compatibility that caused a few more issues, but that's not much to talk about, just proving my point
All I ask is assistance in disabling the unification which seems to not work for me

**Comment by bHXDgiuMa5yLHyC8** [comment]
Did you try setting the TFC items as special unification targets?

**Comment by UnlimatedStone9** [comment]
No I did not actually, I'm not aware of how any such feature works

**Comment by bHXDgiuMa5yLHyC8** [comment]
You go into Unification.cfg and replace every "=false" inside terrafirmacraft { ... } with "=true". This should solve any problems you have with gems or any other TFC item.

**Comment by UnlimatedStone9** [comment]
I assume this disables the unification of all of those items permanently? If so I assume that's all I need to do

**Comment by UnlimatedStone9** [comment]
Sorry to bring this back up again but this appears to not work, I have since set everything to true in the configs and items are still converting over, is there by any chance a single *disableAllConversions=true/false* config option or anything?

**Comment by UnlimatedStone9** [comment]
Perhaps the config file is replaced every time a new mod is installed? if so then it might have overwritten my changes

**Comment by draknyte1** [comment]
Actually, no. Explicit support isn’t given for TFC.. 

Just the materials from it, which aren’t particularly unique to that mod. Bismuth is the worst example because Greg used it in GT, so it’s now unrelated to TFC support.  

![591cb4a2-3300-4a3a-872a-47bf35cbdcd6](https://user-images.githubusercontent.com/3060479/51988338-54efa380-249c-11e9-9022-d1a5d949912c.png)

**Comment by UnlimatedStone9** [comment]
Can we please forget that and get back to the issue at hand?

**Comment by UnlimatedStone9** [comment]
Ok, it appears to me that whatever I do, the config files "Unification.cfg" are getting overwritten every time I launch the game, making them entirely useless and leaving me with this annoying bug permanently
